include/nccl_ofi_rdma.h

```diff
@@ -71,6 +71,49 @@ static_assert(MAX_NUM_RAILS <= UINT16_MAX);
  */
 #define NCCL_OFI_RDMA_SEQ_BITS     (10)
 
+/*
+ * @brief	Message sequence number bitmask for immediate data
+ */
+#define MSG_SEQ_NUM_MASK (((uint64_t)1 << NCCL_OFI_RDMA_SEQ_BITS) - 1)
```

Comment:
I'm really not happy having all this be "public", it feels like we've taken a wrong approach somewhere.